perception/image_projection_based_fusion/docs/pointpainting-fusion.md

```diff
@@ -14,22 +14,22 @@ The lidar points are projected onto the output of an image-only 2d object detect
 
 ### Input
 
-| Name                  | Type                                                     | Description                                                                        |
-| --------------------- | -------------------------------------------------------- | ---------------------------------------------------------------------------------- |
-| `input`               | `sensor_msgs::msg::PointCloud2`                          | input pointcloud                                                                   |
-| `input/camera_infoID` | `sensor_msgs::msg::CameraInfo`                           | camera information to project 3d points onto image planes, `ID` is between 0 and 7 |
-| `input/roisID`        | `tier4_perception_msgs::msg::DetectedObjectsWithFeature` | ROIs from each image, `ID` is between 0 and 7                                      |
-| `input/image_rawID`   | `sensor_msgs::msg::Image`                                | images for visualization, `ID` is between 0 and 7                                  |
+| Name                     | Type                                                     | Description                                               |
+| ------------------------ | -------------------------------------------------------- | --------------------------------------------------------- |
+| `input`                  | `tier4_perception_msgs::msg::DetectedObjectsWithFeature` | clustered pointcloud                                      |
+| `input/camera_info[0-7]` | `sensor_msgs::msg::CameraInfo`                           | camera information to project 3d points onto image planes |
+| `input/rois[0-7]`        | `tier4_perception_msgs::msg::DetectedObjectsWithFeature` | ROIs from each image                                      |
+| `input/image_raw[0-7]`   | `sensor_msgs::msg::Image`                                | images for visualization                                  |
 
 | `
```

Comment:
This is outside the table. Maybe this is a typo?

Reply:
fixed at [16eee84](https://github.com/autowarefoundation/autoware.universe/pull/2178/commits/16eee84b4cc7e04f952a83039507335566cd699b).